presto-native-execution/presto_cpp/main/types/PrestoToVeloxQueryPlan.cpp

```diff
@@ -387,6 +403,52 @@ std::unique_ptr<common::BigintRange> dateRangeToFilter(
   return std::make_unique<common::BigintRange>(low, high, nullAllowed);
 }
 
+std::unique_ptr<common::BigintRange> shortDecimalRangeToFilter(
+    const protocol::Range& range,
+    bool nullAllowed,
+    const VeloxExprConverter& exprConverter,
+    const TypePtr& type) {
+  bool lowUnbounded = range.low.valueBlock == nullptr;
+  auto low = lowUnbounded
+      ? std::numeric_limits<int64_t>::min()
```

Comment:
The numeric limits for decimals and integers are not the same. In this context, just verify if we need to use UnscaledXXXDecimal::min() or max().

Reply:
Updated.